### PR TITLE
add meshmap project card in projects grid

### LINKED ISSUE #3621: [Projects] Add a MeshMap project card
#### Current Behavior
The [projects page](https://layer5.io/projects) has a grid cards that list and link to our various projects. The [MeshMap](https://layer5.io/cloud-native-management/meshmap) project is missing.

#### Implementation
Using the MeshMap logo, add another project card into the mix.

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by avikt18**
**Description**
This PR adds a new card for the project MeshMap in the projects grid on this [webpage](https://layer5.io/projects)

This PR fixes #3621 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5d8c4be921d50253c7e2ffe59310835dc6d30156 at: https://63ca8bfad0155f210ec419b5--layer5.netlify.app

**Comment by avikt18** [comment]
Oops!! The lighthouseCI test is failing :eyes: @leecalcote

**Comment by leecalcote** [comment]
> Oops!! The lighthouseCI test is failing 👀 @leecalcote

@ayushthe1 will be coming around on these items. 👍

File: src/sections/Projects/Project-grid/index.js [hunk 1]
```diff
@@ -69,8 +70,14 @@ const ProjectPage = ({ theme }) => {
                 <h5>Service Mesh Performance</h5>
               </div>
             </Link>
+            <Link to="/projects/nighthawk" className="project__card nine">
```
Comment (leecalcote) [inline src/sections/Projects/Project-grid/index.js:73]:
```suggestion
            <Link to="/cloud-native-management/meshmap" className="project__card nine">
```

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 2a28bb935f14815e46c710774e2d4fe11f4b5979 at: https://63cae1beb4c06915adffd9f2--layer5.netlify.app

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@avikt18 Please use rem instead of px for height, width, and margins...

**Comment by avikt18** [comment]
Ohh..okay @Chadha93

**Comment by Nikhil-Ladha** [comment]
> Ohh..okay @Chadha93. I will fix it for margins. Do you also want me to change image heights and widths to rem? because all the images have it in px

Yes, please take the equal (or closest) value in `rem` and use `rem` for `width`/`height` as well

**Comment by l5io** [comment]
🚀 Preview for commit d285da7a9ee31f51d1a109ff1fef1d0b97b49e82 at: https://63cbf5ccf84df005e683b5c5--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]
Cool :ok_hand:  Thank you @avikt18

**Pull request merged**